populate/src/main/java/bio/terra/pearl/populate/service/BasePopulator.java

```diff
@@ -31,13 +31,14 @@ public T populate(P context, boolean overwrite) {
 
     public T populateFromString(String fileString, P context, boolean overwrite) throws IOException {
         D popDto = readValue(fileString);
-        return populateFromDto(popDto, context, overwrite);
+        return this.populateFromDto(popDto, context, overwrite);
     }
 
     public D readValue(String popDtoString) throws IOException {
         return objectMapper.readValue(popDtoString, getDtoClazz());
     }
 
+    @Transactional
```

Comment:
minor nit: I am not sure we need this, as the parent populate function is Transactional.